packages/ketcher-core/src/domain/entities/image.ts

```diff
@@ -15,16 +15,22 @@
  ***************************************************************************/
 
 import { BaseMicromoleculeEntity } from 'domain/entities/BaseMicromoleculeEntity';
-import { Point, Vec2 } from 'domain/entities/vec2';
+import { Vec2 } from 'domain/entities/vec2';
 import { getNodeWithInvertedYCoord, KetFileNode } from 'domain/serializers';
-import { RASTER_IMAGE_SERIALIZE_KEY } from 'domain/constants';
-
-interface KetFileNodeContent {
-  bitmap: string;
-  halfSize: Point;
+import { IMAGE_SERIALIZE_KEY } from 'domain/constants';
+
+export interface KetFileImageNode extends KetFileNode<string> {
+  format: string;
+  boundingBox: {
+    x: number;
+    y: number;
+    z?: number;
```

Comment:
Do we need z coordinate?

Reply:
We have it by default across the app, whenever we are using Vec2